deploy/018-native-token-gateway.ts

```diff
@@ -13,32 +13,44 @@ interface VTokenConfig {
   address: string;
 }
 
-const VWNativeInfo: { [key: string]: VTokenConfig } = {
-  bsctestnet: {
-    name: "vWBNB_LiquidStakedBNB",
-    address: ilBscTestnet.VToken_vWBNB_LiquidStakedBNB.address,
-  },
-  bscmainnet: {
-    name: "vWBNB_LiquidStakedBNB",
-    address: ilBscMainnet.VToken_vWBNB_LiquidStakedBNB.address,
-  },
-  sepolia: {
-    name: "vWETH_Core",
-    address: ilSepolia.VToken_vWETH_Core.address,
-  },
-  ethereum: {
-    name: "vWETH_Core",
-    address: ilEthereum.VToken_vWETH_Core.address,
-  },
+const VWNativeInfo: { [key: string]: VTokenConfig[] } = {
+  bsctestnet: [
+    {
+      name: "vWBNB_LiquidStakedBNB",
+      address: ilBscTestnet.VToken_vWBNB_LiquidStakedBNB.address,
+    },
+  ],
+  bscmainnet: [
+    {
+      name: "vWBNB_LiquidStakedBNB",
+      address: ilBscMainnet.VToken_vWBNB_LiquidStakedBNB.address,
+    },
+  ],
+  sepolia: [
+    {
+      name: "vWETH_Core",
+      address: ilSepolia.VToken_vWETH_Core.address,
+    },
+    {
+      name: "vWETH_LiquidStakedETH",
+      address: ilSepolia.VToken_vWETH_LiquidStakedETH.address,
+    },
+  ],
+  ethereum: [
+    {
+      name: "vWETH_Core",
+      address: ilEthereum.VToken_vWETH_Core.address,
+    },
```

Comment:
We'll need to add here the vWETH for the LST pool on Ethereum (to be enabled)

Reply:
Also the vWETH on opBNB testnet and mainnet

Reply:
1.  I could not find the vWETH market for the LST pool on Ethereum mentioned in the venus docs [here](https://docs-v4.venus.io/deployed-contracts/isolated-pools#ethereum). But I found it in the deployments folder for Ethereum. I think [this](https://github.com/VenusProtocol/vips/pull/189) vip has not been proposed.

2. It will be vWBNB and for that we will have to change the underlying of vWBNB of core pool on opbnbtest [here](https://testnet.opbnbscan.com/address/0xD36a31AcD3d901AeD998da6E24e848798378474e?tab=Contract&p=1&view=contract_proxyRead)

Reply:
Updated the deployment script: https://github.com/VenusProtocol/isolated-pools/pull/361/commits/2b28492d60df941fdec01c94fc612fc0cd23b42a

Reply:
1. The TX has been executed today: https://etherscan.io/tx/0x6aab7c9472b72b0eae785d452a68806cb93f23105491a6099d6d4fc63ee76ca3
2. you are right, it would be for vWBNB. We have VEN-2433 to complete the change